src/index.js.flow

```diff
@@ -0,0 +1,32 @@
+// @flow strict
+
+export type Version = 0 | 1
+export type Codec = string
+export type Multihash = Buffer
+export type BaseEncodedString = string
+
+declare class CID<a> {
+  constructor(Version, Codec, Multihash): void;
```

Comment:
CID now takes an optional `multibaseName` string as the 4th param as of #77

Reply:
Thanks for pointing that out @olizilla, I'll create a followup pull to add that param.

Reply:
@olizilla @mikeal Can I assume that there is a consensus on wanting to keep the typedefs though ?

Reply:
Not sure if there's consensus, but I'm the lead maintainer and I'd like to keep them :)

Reply:
@vmx I think @mikeal was just looking +1 from some one, and it sounds like we have one. Thanks